src/main/java/Duke/commands/MarkTaskCommand.java

```diff
@@ -0,0 +1,28 @@
+package Duke.commands;
+
+import Duke.exceptions.DukeException;
+import Duke.storage.Storage;
+import Duke.tasks.TaskList;
+import Duke.ui.UI;
+
+public class MarkTaskCommand extends Command {
+
+    private final int taskID;
```

Comment:
I think taskID can be written as taskId instead to conform to the camelCase standard